### PR TITLE
Threejs in react

### PR DESCRIPTION
**Pull request by cb299792458**

### CODE REVIEW

File: frontend/src/pages/TestB.jsx [hunk 1]
```diff
@@ -0,0 +1,118 @@
+import * as THREE from 'three'
+import { OrbitControls } from 'three/examples/jsm/controls/OrbitControls.js'
+import GUI from 'lil-gui'
+import Stats from 'stats.js'
+import { useEffect, useRef } from 'react'
+
+const TestMap = () => {
+    const mapRef = useRef();
+
+    useEffect(() => {
```
Comment (rockokw) [inline frontend/src/pages/TestB.jsx:10]: Don't we need an empty dependency array for this one too?

File: frontend/src/pages/TestC.jsx [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+
+const TestC = () => {
+
+    return (<>
+        <h1>Test Page C</h1>
+    </>)
+}
+
+export default TestC;
```
Comment (rockokw) [inline frontend/src/pages/TestC.jsx:10]: Nit: This and the other `Test*.jsx` files don't have newlines at the end

File: frontend/src/App.jsx [hunk 3]
```diff
@@ -1,21 +0,0 @@
-// import './App.css';
```
Comment (rockokw) [inline frontend/src/App.jsx:1]: Do we need this file anymore?

**Review by rockokw — APPROVED** [review]

File: frontend/src/App.jsx (hunk 3 above)
Comment (cb299792458) [reply frontend/src/App.jsx:1]: Nope, the main game will probably be in the `<HomePage>` component now.

**Pull request merged**